lib/lib/docker_args.coffee

```diff
@@ -30,13 +30,14 @@ formatPortBindings = (ports) ->
 
   for port in ports
     [dst, src] = port.split(':')
+    [src, protocol] = if src? && src.indexOf('/') > 0 then src.split('/') else [src, 'tcp']
```

Comment:
This ordering means that '4444/udp' doesn't work, for the case of binding UDP to a random port.

I like your `indexOf` pattern; do you think that you could use that on line 32 for src / dst, which would mean we could get rid of the awkward `unless src?` statement below?